_posts/2023-11-03-testing-a-foreman-installer-patch-with-packit-and-forklift.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: Testing a foreman-installer patch with Packit and forklift
+date: 2023-11-03 11:41:32
+author: Evgeni Golov
+tags:
+- foreman
+- packit
+---
+
+Today I was working on a potential fix for the issue that [Puppetserver doesn't start when deployed with fapolicyd in enforcing mode](https://github.com/theforeman/foreman-fapolicyd/issues/7).
+
+I knew that the potential fix is to set `PrivateTmp=true` in the systemd unit, so the [patch](https://github.com/theforeman/puppet-puppet/pull/892) was quickly written, but how would one verify it end to end?
+
+<!--more-->
```

Comment:
Does this do anything?

Reply:
the part before more is published via RSS, the rest is not